build.gradle.kts

```
@@ -0,0 +1,65 @@
+plugins {
+    id("com.hivemq.extension")
+    id("com.github.hierynomus.license")
+    id("io.github.sgtsilvio.gradle.defaults")
+    id("org.asciidoctor.jvm.convert")
+}
+
+group = "com.hivemq.extensions"
+description = "HiveMQ Mqtt Message Log Extension"
+
+hivemqExtension {
+    name.set("HiveMQ Mqtt Message Log Extension")
+    author.set("HiveMQ")
+    priority.set(1000)
+    startPriority.set(10000)
```

```suggestion
    startPriority.set(1000)
```
default value is 1000, should stay the same as before